README.md

```diff
@@ -14,8 +14,8 @@ This is a repository intended to serve as a starting point if you want to bootst
 
 ## 🏁 How To Start
 
-1. Install Java 11: `brew cask install corretto` or download it [here](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html)
-2. Set it as your default JVM: `export JAVA_HOME='/Library/Java/JavaVirtualMachines/amazon-corretto-11.jdk/Contents/Home'`
+1. Install Java 17: `brew cask install corretto` or download it [here](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html)
+2. Set it as your default JVM: `export JAVA_HOME='/Library/Java/JavaVirtualMachines/amazon-corretto-17.jdk/Contents/Home'`
```

Comment:
Future improvement: Linux, Windows and Docker setup.

(I don't want to put effort on this right now 😬)